### DISCUSSION TITLE
New version eta

### DISCUSSION THREAD

**Discussion by rohitsaha29** [Alex313031/Thorium-Special#56] [General]
Any idea when 113 chrome/new optimizations Thorium might be released for MacOS ARM?

**Comment by midzer** [comment]
Hi @rohitsaha29

We have issues building M113 for Mac. Maybe we have to skip this version. Gonna let you know here.

**Comment by rohitsaha29** [comment]
@midzer damn that’s unfortunate hope you guys can figure it out. Thanks for the updates!

**Comment by Alex313031** [comment]
@rohitsaha29 We built 113 now.

**Comment by rohitsaha29** [comment]
@Alex313031 are we getting a 114 for ARM M1 MacOS?

**Reply by gz83** [reply]
We will release new versions periodically, please wait patiently for the new version to be released, do not rush

@rohitsaha29

**Reply by rohitsaha29** [reply]
@gz83 I understand, I was just curious(bc I saw linux release but no MacOS), no rush
